community/teams/cuda.tt

```diff
@@ -0,0 +1,61 @@
+[% WRAPPER layout.tt title="CUDA Team" handlesLayout=1 %]
+
+<div class="page-title">
+  <div>
+    <a href="[% root%]community/index.html#governance-teams">Community</a>
+    <span>→</span>
+  </div>
+  <h1>CUDA team</h1>
+</div>
+
+<section class="governance-description">
+  The CUDA team is responsible for all things CUDA-related in <a href="https://nixpkgs.org">Nixpkgs</a>.
+</section>
+
+<section class="governance-team">
+  <aside>
+    <h2>Members</h2>
+    <ul>
+      <li>
+        Connor Baker (<a href="https://discourse.nixos.org/u/ConnorBaker">@ConnorBaker</a>)<br/>
+        CUDA maintainer, sponsored by <a href="https://tweag.io">Tweag</a>
+      </li>
+    </ul>
+
+    <h2>Contact</h2>
+    <ul>
+      <li><a href="https://discourse.nixos.org/t/announcing-the-nixos-cuda-maintainers-team-and-a-call-for-maintainers/18074">Discourse</a></li>
+      <li><a href="https://app.element.io/#/room/#cuda:nixos.org">Matrix</a></li>
+    </ul>
+  </aside>
+
+  <h2>Goals</h2>
+  <ul>
+    <li>
+      support community efforts involving packaging or distributing CUDA-enabled software
+    </li>
+    <li>
+      document CUDA packaging best practices, guidelines, and patterns
+    </li>
+    <li>
+      improve the quality of CUDA packaging
+    </li>
+    <li>
+      make Nix/NixOS the first choice for users of CUDA-related software, especially in machine learning and scientific computing
```

Comment:
Suggestion: order this by specificity. What you really care about and can work towards should be first. The last item is not really a goal but a recurrent task, but that is fine.

```suggestion
      make Nix and NixOS the first choice for users of CUDA-related software, especially in machine learning and scientific computing
    </li>
    <li>
      increase the number and improve the quality of CUDA packages
    </li>
    <li>
      document CUDA packaging best practices, guidelines, and patterns
    </li>
    <li>
      support community efforts involving packaging or distributing CUDA-enabled software
```

Reply:
One could change the list heading to "Goals and responsibilities", but as said, it's fine as it is. You can (and very probably will) refine it as you go.

Reply:
Fixed in newest